packages/opentelemetry-tracing/src/BasicTracerProvider.ts

```diff
@@ -47,6 +47,9 @@ export class BasicTracerProvider implements api.TracerProvider {
       logger: this.logger,
       resource: this.resource,
     });
+    if (this._config['gracefulShutdown']) {
```

Comment:
why bracket notation?

Reply:
I usually program in C/C++, I am not attached to any one notation and if the project does not use this I have no issue switching. Thanks for bringing this to my attention!

Reply:
typically the bracket notation in typescript means you are accessing a private property